.github/workflows/dev_pr.yml

```diff
@@ -80,5 +80,9 @@ jobs:
         if: '! github.event.pull_request.draft'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        permissions:
+          contents: read
+          issues: write
+          pull-requests: write
```

Comment:
@jbonofre Can we use this configuration? Is this still satisfied our policy? https://infra.apache.org/github-actions-policy.html

This is what I asked on Zulip: [#java-chat > GitHub Action versions alias @ 💬](https://ursalabs.zulipchat.com/#narrow/channel/483042-java-chat/topic/GitHub.20Action.20versions.20alias/near/503947400)

Reply:
Yes, it should not be a problem to write issues/pull-requests.

Do you really need to have `GH_TOKEN` env variable ? Why not directly using `GITHUB_TOKEN` ?

Reply:
That's my point (sorry if I wasn't clear): why storing as env variable ? 

I discussed with Gavin (from the ASF Infra) to clarify the "triggers" statement on the GitHub Action policy page.

Reply:
Because the script is invoking the GitHub CLI to do things, and the GitHub CLI needs a token from an environment variable

Reply:
@jbonofre was there any clarification here?

Reply:
@lidavidm let me double check with the Infra again (sorry I forgot).

Reply:
Since **pull_request_target** runs in the target repository's context with write access to secrets, directly executing a .sh file from a PR poses a security risk. To ensure the script hasn't been tampered with, we must verify its SHA256 checksum before execution to prevent unintended scripts from running and potential malicious attacks.

Reply:
This should be from the main branch, though, not from the PR. (But I guess would it be clearer/safer to have a separate repo of custom actions for the project that we can use and pin?)

Reply:
oops...we're not executing the checkout operation... please disregard me

Reply:
I double checked and it looks good to me.